wd_capabilities.au3

```diff
@@ -264,7 +263,7 @@ EndFunc   ;==>_WD_CapabilitiesGet
 ;                  $s_BrowserType      - [optional] a string value. Default is ''. One of:
 ;                               | 'chrome'
 ;                               | 'firefox'
-;                               | 'edge'
+;                               | 'msedge'
```

Comment:
This is where I was suggesting that you could simplify the entry. Either change it to something like 

`$s_BrowserType      - [optional] The browser name. Default is ''`

(without the list of valid options) or you'll have to add `Opera` to the list of valid entries.